marklogic-data-hub-central/ui/src/components/advanced-settings/advanced-settings-dialog.tsx

```diff
@@ -56,7 +55,10 @@ const ActivitySettingsDialog = (props) => {
 
   const tgtDatabaseOptions = ['data-hub-STAGING','data-hub-FINAL'];
 
-  const provGranOptions = ['coarse', 'off'];
+  const provGranOptions = {
```

Comment:
Same thing here - let's do provenanceGranularityOptions. A benefit - let's say someone wants to search the codebase for all references to granularity options. One approach would be to search for "granularity", ignoring case. That search won't find this code though, which could lead to a mistake being made. But if we use the same term throughout the codebase, it's easy to understand and easy to search for references.

Reply:
Done.